models/index.js

```
@@ -1,5 +1,9 @@
 'use strict';
+// I noticed that you're using 'use strict' here, but not
+// other places?
 
```

I'm using strict on all the models (index.js, todo.js and todo_list.js) to prevent silent errors. I add it in those files because that's where most of the "custom" code would go.